extensions/kafka-client/runtime/src/main/java/io/smallrye/reactive/kafka/graal/StrimziSubstitutions.java

```diff
@@ -72,10 +72,8 @@ private static boolean isJson(Object o) {
 
 @TargetClass(className = "com.jayway.jsonpath.internal.DefaultsImpl", onlyWith = HasStrimzi.class)
 final class Target_com_jayway_jsonpath_internal_DefaultsImpl {
-
-    @RecomputeFieldValue(kind = RecomputeFieldValue.Kind.FromAlias)
-    @Alias
-    public static Target_com_jayway_jsonpath_internal_DefaultsImpl INSTANCE = new Target_com_jayway_jsonpath_internal_DefaultsImpl();
```

Comment:
@ozangunalp do you recall why you needed this field recomputation?

Reply:
My thinking was that because `DefaultsImpl` holds a field value of `JsonSmartMappingProvider`, recalculating the static instance would be needed to remove any instance of `JsonSmartMappingProvider` from the native image.

Reply:
But now that I re-run with or without I don't see any difference.

Reply:
Good, the deletion of the `mappingProvider` field in this PR should take care of your initial concern as well then.

Reply:
Indeed, Thanks!